setup.py

```diff
@@ -1,12 +1,19 @@
+import os
+
 from setuptools import find_packages, setup
 
 
+# Load README
 with open('README.md') as f:
     readme = f.read()
 
+# Load version number
+with open(os.path.join('cellmlmanip', 'version.txt'), 'r') as f:
```

Comment:
Have you checked that the version.txt file is installed as part of cellmlmanip?

Reply:
uups now I have!